manifests/base/templates/telemetryv2_1.4.yaml

```diff
@@ -1,4 +1,3 @@
-{{ if or (eq .Values.revision "") (not .Values.clusterResources) }}
```

Comment:
This is a bit tricky - we don't want to install this for each control plane instance. It'll not work with helm3 for example. 

I would expect we'll still have a 'default' or 'core' control plane - with revision == "" or "default", and we can install Istio CRs that are common with that resource.

Or we can target the EnvoyFilter to the revision, and use name suffix. That may be even better.

Reply:
But please change this before merging - it would break helm3. 

Sorry for not adding the tests to the suite - didn't have time to change the base template. The makefile should have tests to install 2 revisions with helm3.

Reply:
This is moved to the base install, so it should only be applied once when you are doing the "default" install, right? Or will it break in some other way I am missing? 

Let me try out the helm3 stuff

Reply:
I'm not sure - now we'll have CRD and CR in the same place. 
The base install is not supposed to install user CRs - just cluster wide stuff ( CRD, ClusterBindings, etc). And some exceptions - but I don't think this qualifies. It's just a hack - using the installer to implement a code feature. 

I have no reason to believe this will stay common for all versions of Istio - i.e. in future we'll not have some better way to configure.

Reply:
I guess we could try it - it's still a hack in either place... But I'm hoping base will be cleaner and independent of revision, and you know how precendent works...

Reply:
Ack. For now ill revert this part and think about the best way to manage these envoyfilters

Reply:
Actually - I thought more, and it's really not a good idea. What if I wan to install Istio with telemetry completely off ?  I would install base, and an Istiod revision with the telemetry off - but because the base already installed the filters - it won't work.

The filters were installed with the 'default' revision, because the default has telemetry on. 
But there is no requirement that user installs the default. They may also want to customize what filters they apply - after all EnvoyFilter is a user resource. We install a default in the default revision, but it doesn't look like it must be hardcoded for everyone.